tests/unit/test_preprocessing.py

```diff
@@ -23,3 +24,16 @@ def test_loner_removal(self):
         num_vertices = len(g.vertices)
         g = preprocessing.remove_loners(g)
         self.assertEqual(num_vertices - 1, len(g.vertices))
+
+    def test_use_complement(self):
+        # A nice star shaped graph that should use the complement
+        many_edges_graph = create_graph_helper([(0, 3), (1, 2), (1, 3), (1, 4), (2, 3), (2, 4), (3, 4)])
+        g, h = preprocessing.check_complement(many_edges_graph, many_edges_graph)
+        self.assertTrue(g is not many_edges_graph)
```

Comment:
je kunt ook checken dat het complement `(n*(n-1)/2) - aantal edges in g` edges heeft